pkgs/applications/misc/polychromatic/default.nix

```
@@ -63,7 +64,7 @@ python3Packages.buildPythonApplication rec {
   ];
 
   dontWrapGapps = true;
-  dontWrapQtApps = true;
+  dontWrapQtApps = false;
 
   makeWrapperArgs = [
     "\${gappsWrapperArgs[@]}"
```

```suggestion
    "\${gappsWrapperArgs[@]}"
    "\${qtWrapperArgs[@]}"
```
we want to add it here to avoid the double wrapping.

---

Carried that over to #202908 because that looks broken to me.